src/controller/CHIPDeviceController.h

```diff
@@ -397,13 +397,13 @@ class DeviceCommissionerRendezvousAdvertisementDelegate : public RendezvousAdver
      * @brief
      *   Starts advertisement of the device for rendezvous availability.
      */
-    CHIP_ERROR StartAdvertisement() const override { return CHIP_NO_ERROR; }
```

Comment:
Why remove `const`?

Reply:
The function `CommissionManager::StartAdvertisement()` is modifying the interal states `mCommissioningWindowOpen`.
Previously it's modifying a global variable `gPairingWindowOpen` in `Server.cpp` and sneaked out of the const check. It's never const so I removed it for refactoring.